troi/core.py

```diff
@@ -9,123 +9,6 @@
 import troi.utils
 from troi.patch import Patch
 
-default_patch_args = dict(
-    debug=False,
-    echo=True,
-    save=False,
-    token=None,
-    upload=False,
-    args=None,
-    created_for=None,
-    name=None,
-    desc=None,
-    min_recordings=10,
-    spotify=None
-)
-
-
-def generate_playlist(patch: Patch, args: Dict):
```

Comment:
We use generate_playlist in LB so need to carefully update it next time we update troi there.